scripts/irods/test/test_icommands_file_operations.py

```diff
@@ -535,6 +555,23 @@ def test_irsync_r_nested_coll_to_coll(self):
                         msg="Files missing from vault:\n" + str(local_files - files_in_vault) + "\n\n" +
                             "Extra files in vault:\n" + str(files_in_vault - local_files))
 
+@unittest.skipIf(test.settings.TOPOLOGY_FROM_RESOURCE_SERVER, "Skip for topology testing from resource server")
+class Test_ICommands_File_Operations_3(resource_suite.ResourceBase, unittest.TestCase):
+    plugin_name = IrodsConfig().default_rule_engine_plugin
+    class_name = 'Test_ICommands_File_Operations_3'
+
+    def setUp(self):
+        # TODO: Remove this sleep when #4359 is resolved
+        time.sleep(30)
```

Comment:
note that https://github.com/irods/irods/issues/4359 is closed.    investigate removing this sleep (and others in this file).

Reply:
Yes. That was part of the plan too.

I'm going to address that in another commit or separate PR.

Reply:
Removing the call to sleep did not break anything. All of the tests passed.

Reply:
glory be.